.azure_pipelines_snp.yml

```diff
@@ -33,28 +33,47 @@ jobs:
       vmImage: ubuntu-20.04
     steps:
       - script: |
+          echo -e "\nGenerating an SSH key for use with the executors..."
+          ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa -N ""
+          PUB_KEY=`cat ~/.ssh/id_rsa.pub`
+          PRIV_KEY=`cat ~/.ssh/id_rsa | tr '\n' '?'`
```

Comment:
This replacement of newlines is because azure pipelines can't handle multi-line variables, my approach of replacing them with `?` is probably dodgy as I think they might be able to appear in the private key? So open to suggestions of other characters to use or other approaches entirely

Reply:
Amaury helpfully suggested base64'ing it instead which makes much more sense, so I'll do that

Reply:
Addressed in https://github.com/microsoft/CCF/pull/4657/commits/8b4e0425f11f1e47f0d4663e21537b1e7f36e951